<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreerCompte</class>
 <widget class="QDialog" name="CreerCompte">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>834</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 20pt &quot;Malgun Gothic&quot;;
color:rgb(0, 0, 0)</string>
   </property>
   <property name="text">
    <string>Creation du compte</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>130</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 12pt &quot;Segoe UI&quot;;
color:rgb(0, 0, 0)</string>
   </property>
   <property name="text">
    <string>Nom :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>190</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 12pt &quot;Segoe UI&quot;;
color:rgb(0, 0, 0)</string>
   </property>
   <property name="text">
    <string>Pr√©nom :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>250</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 12pt &quot;Segoe UI&quot;;
color:rgb(0, 0, 0)</string>
   </property>
   <property name="text">
    <string>Mot De Passe :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>60</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 12pt &quot;Segoe UI&quot;;
color:rgb(0, 0, 0)</string>
   </property>
   <property name="text">
    <string>Identifiant :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="NomLine">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>120</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="PrenomLine">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>180</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="MdpLine">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>240</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="IdLine">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>60</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>430</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Valider</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>430</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Annuler</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>310</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 12pt &quot;Segoe UI&quot;;
color:rgb(0, 0, 0)</string>
   </property>
   <property name="text">
    <string>Confirmez :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ConfirmezLine">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>300</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>370</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 12pt &quot;Segoe UI&quot;;
color:rgb(0, 0, 0)</string>
   </property>
   <property name="text">
    <string>Nom de compte :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ndcLine">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>360</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
